docs/docs/components.mdx

```diff
@@ -1572,7 +1572,7 @@ See [starspace paper](https://arxiv.org/abs/1709.03856) for details.
 * **Short**
 
   Classifies a message with the intent `nlu_fallback` if the NLU intent classification
-  scores are ambiguous.
+  scores are ambiguous. The confidence is set to 1 - top intent confidence.
```

Comment:
```suggestion
  scores are ambiguous. The confidence is set to `1 - top intent` confidence.
```

Reply:
then it should be `1 - top intent confidence` but that's too much for ``, no?

Reply:
I'd put it all in '`' but up to you